src/components/NavBar.js

```diff
@@ -1,31 +1,51 @@
 import React, { Component } from 'react';
 import { withRouter, Link } from 'react-router-dom';
-import { Navbar } from 'react-bootstrap';
+import { Navbar, Nav, NavLink } from 'react-bootstrap';
 import ResponsiveMenu from 'react-responsive-navbar';
 import { FaBars, FaTimes } from 'react-icons/fa';
 import Button from '@material-ui/core/Button';
+import Input from '@material-ui/core/Input';
 import Image from 'react-bootstrap/Image';
 import TextField from '@material-ui/core/TextField';
+import InputBase from '@material-ui/core/InputBase';
+import InputAdornment from '@material-ui/core/InputAdornment';
 import Autocomplete from '@material-ui/lab/Autocomplete';
 import AccountCircle from '@material-ui/icons/AccountCircle';
+import SearchIcon from '@material-ui/icons/Search';
 import Dialog from '@material-ui/core/Dialog';
 import Amplify, { Auth, API, graphqlOperation } from 'aws-amplify';
 import { Authenticator, Greetings } from 'aws-amplify-react';
-import './NavBar.css';
+//import './NavBar.css';
```

Comment:
The navigation bar is not responsive at all. The page looks really bad on mobile, it was fine before

Reply:
Need to add a Nav.Collapse, and probably add the discussion board as a menu item